pyproject.toml

```diff
@@ -48,13 +48,15 @@ pre-commit = "^2.13.0"
 pytest-aiohttp = "^1.0.4"
 time-machine = "^2.6.0"
 Jinja2 = "<3.1.0"
+syrupy = "^3.0.0"
```

Comment:
This is the snapshot testing library we're wrapping which handles associating pytest test functions with snapshots on disk.